### PR TITLE
[WIP] Documentation

### PR DESCRIPTION
**Pull request by mdkrol**

### CODE REVIEW

**Comment by mdkrol** [comment]
Hoi Casper,

Met deze doc updates de wens om 2 parameters te hernoemen.
In misc.Step de parameter location naar value
in temporal.Cumulative de parameter frequency naar period

File: dask_geomodeling/raster/spatial.py [hunk 1]
```diff
@@ -102,14 +102,20 @@ def expand_request_meters(request, radius_m=1):
...
+    
+    Args:
+      store (RasterBlock): Raster to perform dilation on
+      values (list): Only cells with these values are dilated
+
+    Returns:
+      Rasterblock where cells in values list are dilated
+    
+    See also:
+      https://homepages.inf.ed.ac.uk/rbf/HIPR2/dilate.htm
```
Comment (caspervdw) [inline dask_geomodeling/raster/spatial.py]: Don't link to sites like this, this like has a high chance to become unavailable.
Wiki is better: https://en.wikipedia.org/wiki/Dilation_%28morphology%29

File: dask_geomodeling/raster/spatial.py [hunk 2]
```diff
@@ -198,21 +205,20 @@ def process(data, size=None):
 class Smooth(BaseSingle):
     """
     Smooth the values from a raster spatially using gaussian smoothing.
+    
+    Performs a spatial smoothing on the input raster. Uses a Gaussian smoothing algorithm. 
+    
+    Args:
+      store (RasterBlock): Raster to be smoothed
+      size (scalar): Input parameter for the smoothing algorithm. The 'sigma' value for the Gaussian kernal equals ``size / 3``
```
Comment (caspervdw) [inline dask_geomodeling/raster/spatial.py]: The term 'scalar' should become 'number' (maybe also on other places)

File: dask_geomodeling/raster/spatial.py [hunk 3]
```diff
@@ -198,21 +205,20 @@ def process(data, size=None):
...
+      fill (scalar): Value used for 'no data' values in the smoothing process. Defaults to 0. 
+
+    Returns:
+      Smoothed raster
+      
+    See Also:
+      https://homepages.inf.ed.ac.uk/rbf/HIPR2/gsmooth.htm
```
Comment (caspervdw) [inline dask_geomodeling/raster/spatial.py]: Same remark. Use wiki here.

**Comment by mdkrol** [comment]
Hi Casper,

We're doing this in English I guess :)
I updated some stuff after you edits. 
Did you read my previous comment about changing vairable names?
--
In misc.Step de parameter location naar value
in temporal.Cumulative de parameter frequency naar period

**Comment by caspervdw** [comment]
Hi,
The parameter renamings are OK. One thing is that the name `frequency` is also present in the Lizard codebase so that has to be changed also.

Maybe also do the same change in the TemporalAggregate (frequency -> period)?

**Comment by caspervdw** [comment]
Closing in favour of #30

**Pull request closed without merging**